test/Sema/generic-subscript.swift

```diff
@@ -0,0 +1,44 @@
+// RUN: %target-typecheck-verify-swift
+
+
+protocol P
```

Comment:
I just noticed the formatting in this file is a bit inconsistent. Do you mind fixing it?